lumigator/python/mzai/sdk/tests/test_health.py

```diff
@@ -0,0 +1,105 @@
+import importlib.resources
+import json
+from http import HTTPStatus
+from pathlib import Path
+
+from pytest import raises
+from requests.exceptions import HTTPError
+
+
+def test_sdk_healthcheck_ok(mock_requests_response, mock_requests, lumi_client):
+    deployment_type = "local"
+    status = "ok"
+    mock_requests_response.status_code = 200
+    mock_requests_response.json = lambda: json.loads(
```

Comment:
Are we testing `mock_requests_response.json` in this method?

Reply:
We're feeding the json that we would get form the server (.json() is used in the request from requests to get the content as a json, so we mock it up)